packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -536,3 +560,86 @@ class VideoCreationOptions {
   /// The type of view to be used for displaying the video player
   final VideoViewType viewType;
 }
+
+/// Represents an audio track in a video with its metadata.
+@immutable
+class VideoAudioTrack {
+  /// Constructs an instance of [VideoAudioTrack].
+  const VideoAudioTrack({
+    required this.id,
+    required this.label,
+    required this.language,
+    required this.isSelected,
+    this.bitrate,
+    this.sampleRate,
+    this.channelCount,
+    this.codec,
+  });
+
+  /// Unique identifier for the audio track.
+  final String id;
+
+  /// Human-readable label for the track.
+  final String label;
```

Comment:
Same question here as for `language`; this can be null for at least ExoPlayer; why force that information to be discarded at this layer?